extensions/typescript/package.nls.json

```diff
@@ -32,5 +32,6 @@
 	"typescript.implementationsCodeLens.enabled": "Enable/disable implementations CodeLens. Requires TypeScript >= 2.2.0.",
 	"typescript.openTsServerLog.title": "Open TS Server log file",
 	"typescript.selectTypeScriptVersion.title": "Select TypeScript Version",
-	"jsDocCompletion.enabled": "Enable/disable auto JSDoc comments"
+	"jsDocCompletion.enabled": "Enable/disable auto JSDoc comments",
+	"javascript.implicitProjectConfig.checkJs": "Enable/disable type checking in loose JavaScript files that are not included in any jsconfig or tsconfig projects. Requires TypeScript >= 2.3.1."
```

Comment:
I'm still not sure how to express when this setting applies well. The setting applies to only when a javascript file is not included in a JavaScript project as defined in a `jsconfig` or `tsconfig`. If the file is part of a project, the user must instead enable `checkJs` in the config file itself

@kieferrm Let me know if you have any suggestions here